ci/pipeline/determine_tests_to_run.py

```diff
@@ -138,7 +138,6 @@ def get_commit_range():
         # End of dry run.
 
         skip_prefix_list = [
-            ".buildkite/",
```

Comment:
why is this line remove?

Reply:
Files in this list are basically skipped when determining which tests to run. Without this change, this PR does not trigger rllib tests, even though it changes the rllib.rayci.yaml file :)